playbooks/recreate-etcd-cluster.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+# This playbook forces etcd to recreate cluster memembership allowing new memembers
+# to join the cluster.
+# 
+# WARNING: this will delete all information stored in etcd for the entire cluster.
+
+- include: "{{ playbook_dir }}/check-requirements.yml"
+
+- hosts: all
+  gather_facts: no
+  tasks:
+    - name: Stop etcd
+      sudo: yes
+      service:
+        name: etcd
+        state: stopped
+    - name: Delete etcd data directory
+      sudo: yes
+      shell: rm -rf /var/lib/etcd/*
```

Comment:
Could we variablize this? Looks like the etcd role doesn't necessarily drop it's data in `/var/lib/etcd/`: https://github.com/CiscoCloud/mantl/blob/25a7e6c0b135ee99e9ca6037f8fad7e9cc44487d/roles/etcd/defaults/main.yml#L27